aawordsearch.c

```diff
@@ -646,9 +668,11 @@ main (int argc, char **argv)
             HOST);
     int t = 0;
     int r;
+    if (lang == NULL)
```

Comment:
Need to be checking for incorrect language specifications. I'm thinking a struct with the available choices, similar to this: https://github.com/theimpossibleastronaut/rmw/blob/35ce9f61cc95ab7dcd266b5f3af1d7b458d67155/src/config_rmw.c#L124-L133